### PR TITLE
Fix tab location on firefox nightly

### LINKED ISSUE #568: Tabs are placed in headerbar even when not enabled in Nightly 112
**Describe the bug**

Tabs are placed in headerbar even when not enabled in Nightly 112

**To Reproduce**

1. Use Nightly 112.0a1 (2023-03-12) 
2. See tabs are placed in headerbar

**Expected behavior**

Tabs are placed below URL bar

**Screenshots**

Screenshot from 2023-03-13 21-25-06

**Setup information**
 - OS: Arch Linux
 - Installation method: Flatpak
 - Firefox version: 112.0a1 (2023-03-12)
 - Theme version: master 18a4d21238be0adb941ad94d6ae6df5f16bfc5ca
 - Optional features enabled:
 
```
gnomeTheme.hideSingleTab
gnomeTheme.activeTabContrast
gnomeTheme.hideWebrtcIndicator
```

**Additional context**

Working correctly in 110.0.1

Screenshot from 2023-03-13 21-29-28

### PR DESCRIPTION
**Pull request by Titaniumtown**
This PR resolves #568 

I did some digging and figured it out. This seems to work.

Let me know if there's some major glaring issue with it because I know nothing about theming haha.

### CODE REVIEW

**Comment by Titaniumtown** [comment]
seems now there's weird tab title alignment, but idk
image

**Review by rafaelmardojai — CHANGES_REQUESTED** [review]
You must add this change to `toolbox.css` instead of `tabsbar.css`. There's where the toolbars order lives, also there you will find the `gnomeTheme.bookmarksToolbarUnderTabs` part.

**Comment by Titaniumtown** [comment]
Oh ok. Thanks.

**Comment by Titaniumtown** [comment]
I got everything to work now except `tabsAsHeaderbar` @rafaelmardojai could you take a look and see what I've missed?

File: theme/parts/toolbox.css [hunk 1]
```diff
@@ -61,13 +61,16 @@ findbar:-moz-window-inactive description,
 
 /* Reorder toolbars */
 #navigator-toolbox #nav-bar {
-	-moz-box-ordinal-group: 0;
+	-moz-box-ordinal-group: 98;
+	order: 98;
```
Comment (rafaelmardojai) [inline theme/parts/toolbox.css:65]: Is the `order` property needed? I mean, we only target one Firefox version at a time, so we should only keep one property, whatever works.

Comment (Titaniumtown) [reply theme/parts/toolbox.css:65]: It didn't work if `order` was not including. I was constantly messing around with it to work and that's the only way it had the intended result.

Comment (rafaelmardojai) [reply theme/parts/toolbox.css:65]: And if you remove `-moz-box-ordinal-group` it also doesn't work?

Comment (Titaniumtown) [reply theme/parts/toolbox.css:65]: Without `-moz-box-ordinal-group`, `tabsAsHeaderbar` still doesn't work. But the tabs are located beneath the urlbar.

Comment (rafaelmardojai) [reply theme/parts/toolbox.css:65]: I mean if `order` works and `-moz-box-ordinal-group` doesn't, then let's remove  `-moz-box-ordinal-group`  everywhere.

Comment (rafaelmardojai) [reply theme/parts/toolbox.css]: `tabsAsHeaderbar` could be worked later, I can ask the contributor of that feature to take care of it.

Btw have you tried using `!important`?

Comment (Titaniumtown) [reply theme/parts/toolbox.css:65]: > Btw have you tried using !important?

With `-moz-box-ordinal-group`? It did not seem to work, no. I am not too knowledgeable with Firefox development, maybe this is a recent change. I have no idea.

Comment (Titaniumtown) [reply theme/parts/toolbox.css:64]: > `tabsAsHeaderbar` could be worked later, I can ask the contributor of that feature to take care of it.

Ok, sounds good!

Comment (rafaelmardojai) [reply theme/parts/toolbox.css:64]: > > Btw have you tried using !important?
> 
> With `-moz-box-ordinal-group`? It did not seem to work, no. I am not too knowledgeable with Firefox development, maybe this is a recent change. I have no idea.

Yeah, they probably removed their custom prop. Looks good then.

**Review by rafaelmardojai — APPROVED** [review]

**Comment by rafaelmardojai** [comment]
I will be merging this against `beta`.

**Comment by rafaelmardojai** [comment]
Thanks for taking the time btw! Really appreciated.

**Comment by Titaniumtown** [comment]
@rafaelmardojai no problem! Happy to contribute.

**Comment by githubtefo** [comment]
Just update to v113 on Ubuntu 22.04 and the tabsbar are now placed in headerbar as described in the issue. I installed the last stable version of the theme but the problem persists. What am I missing here?

**Pull request merged**